CLAUDE.md

```
@@ -0,0 +1,141 @@
+- Coverage objective is **90% of testable features**.
+- Unit tests are expected for: entity/component manipulation, system and resource containers, system scheduling, resource load/unload, and all non-graphical, non-audio plugins (physics, scripting, scene, utils).
+- Graphics and sound plugins are **exempt from unit tests** — they are covered by functional tests (an example that renders/plays and is run in CI) instead.
+- Functional tests are also expected for plugin creation + binding into the Core, physics simulation accuracy, and script execution.
+
+## Conventions
+
+### Commits and PRs
+
+Angular convention, CI-enforced for both commit titles and PR titles (`.github/workflows/commit_norm_check.yml`, `.github/scripts/pr_title_check.py`):
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)